sklearn/linear_model/_cd_fast.pyx

```diff
@@ -259,14 +259,14 @@ def enet_coordinate_descent(floating[::1] w,
 
     return w, gap, tol, n_iter + 1
 
-
+@cython.boundscheck(False)
```

Comment:
This is set to false by default:

https://github.com/scikit-learn/scikit-learn/blob/e2d120981fa8e520a8bec85e9902530b0c7c15b5/sklearn/_build_utils/__init__.py#L82

```suggestion
```